types/indefinite/index.d.ts

```diff
@@ -8,4 +8,4 @@ export interface Options {
     caseInsensitive?: boolean;
     numbers?: "colloquial";
 }
-export function indefinite(word: string | number, opts?: Options): string;
+export default function indefinite(word: string | number, opts?: Options): string;
```

Comment:
This is not correct, this should be `export = ...`
https://github.com/tandrewnichols/indefinite/blob/master/lib/indefinite.js#L23

Reply:
With the previous code you get a "TypeError: Object(...) is not a function" error.
With the new code, it works perfectly. Note that you should import it as follows:
`import indefinite from "indefinite";`

Reply:
Not resolved, github is hiding this comment incorrectly.